lib/ansible/modules/network/netvisor/_pn_cluster.py

```diff
@@ -29,6 +29,10 @@
 author: "Pluribus Networks (@amitsi)"
 version_added: "2.2"
 short_description: CLI command to create/delete a cluster.
+deprecated:
+  removed_in: '2.8'
```

Comment:
According to [this](https://docs.ansible.com/ansible/2.5/reference_appendices/release_and_maintenance.html#deprecation-cycle), `removed_in` should be current version of Ansible + 4 feature releases. That is `2.12` in this case. If you change 2.8 to 2.12 the error reported by Ansibot will go away. Let me know if you need any help.